### PR TITLE
Shortcuts fix

### LINKED ISSUE #1008: Video player shortcuts work only when player is focused
If I click anywhere outside of the video player and try to use one of the shortcuts, it doesn't work. I imagine they should work anywhere on the page with the video

### LINKED ISSUE #1009: Video page blocks all keyboard interactions
Seems we try to capture to much of keyboard interactions. When I open a tab with the video page, I can no longer use some of my browser shortcuts. For example Cmd+1 to jump to the first tab or Cmd+L to jump to the address bar.

### PR DESCRIPTION
**Pull request by drillprop**
Fixes #1008
Fixes #1009

### CODE REVIEW

File: src/shared/components/VideoPlayer/VideoPlayer.tsx [hunk 1]
```diff
@@ -87,6 +88,31 @@ const VideoPlayerComponent: React.ForwardRefRenderFunction<HTMLVideoElement, Vid
 
   const displayPlayOverlay = playOverlayVisible && !isInBackground
 
+  // handle hotkeys
+  useEffect(() => {
+    if (!player || isInBackground) {
+      return
+    }
+
+    const handler = (e: KeyboardEvent) => {
```
Comment (rafalpawlow) [inline src/shared/components/VideoPlayer/VideoPlayer.tsx]: Meaningful names please 🙂
```suggestion
    const handler = (event: KeyboardEvent) => {
```

File: src/shared/components/VideoPlayer/VideoPlayer.tsx [hunk 2]
```diff
@@ -87,6 +88,31 @@ const VideoPlayerComponent: React.ForwardRefRenderFunction<HTMLVideoElement, Vid
...
+      if (
+        (document.activeElement?.tagName === 'BUTTON' && e.key === ' ') ||
+        document.activeElement?.tagName === 'INPUT'
+      ) {
+        return
+      }
+
+      const playerReservedKeys = ['k', ' ', 'ArrowLeft', 'ArrowRight', 'j', 'l', 'ArrowUp', 'ArrowDown', 'm', 'f']
+      if (playerReservedKeys.includes(e.key)) {
```
Comment (rafalpawlow) [inline src/shared/components/VideoPlayer/VideoPlayer.tsx]: As above

**Review by rafalpawlow — APPROVED** [review]
Looks good, but I added 2 less significant changes

File: src/shared/components/VideoPlayer/VideoPlayer.tsx (hunk 2 above)
Comment (drillprop) [inline src/shared/components/VideoPlayer/VideoPlayer.tsx]:
```suggestion
      if (playerReservedKeys.includes(event.key)) {
```

File: src/shared/components/VideoPlayer/VideoPlayer.tsx [hunk 3]
```diff
@@ -87,6 +88,31 @@ const VideoPlayerComponent: React.ForwardRefRenderFunction<HTMLVideoElement, Vid
...
+        e.preventDefault()
+        hotkeysHandler(e, player)
```
Comment (drillprop) [inline src/shared/components/VideoPlayer/VideoPlayer.tsx]:
```suggestion
        event.preventDefault()
        hotkeysHandler(event, player)
```

File: src/shared/components/VideoPlayer/VideoPlayer.tsx [hunk 4]
```diff
@@ -87,6 +88,31 @@ const VideoPlayerComponent: React.ForwardRefRenderFunction<HTMLVideoElement, Vid
...
 
+  // handle hotkeys
+  useEffect(() => {
+    if (!player || isInBackground) {
+      return
+    }
+
+    const handler = (event: KeyboardEvent) => {
+      if (
+        (document.activeElement?.tagName === 'BUTTON' && e.key === ' ') ||
```
Comment (drillprop) [inline src/shared/components/VideoPlayer/VideoPlayer.tsx]:
```suggestion
        (document.activeElement?.tagName === 'BUTTON' && event.key === ' ') ||
```

File: src/shared/components/VideoPlayer/VideoPlayer.tsx [hunk 5]
```diff
@@ -87,6 +88,31 @@ const VideoPlayerComponent: React.ForwardRefRenderFunction<HTMLVideoElement, Vid
...
+      return
+    }
+
+    const handler = (event: KeyboardEvent) => {
+      if (
+        (document.activeElement?.tagName === 'BUTTON' && event.key === ' ') ||
+        document.activeElement?.tagName === 'INPUT'
+      ) {
+        return
+      }
```
Comment (kdembler) [inline src/shared/components/VideoPlayer/VideoPlayer.tsx:91]: great stuff

**Review by kdembler — APPROVED** [review]
works really well now!

**Pull request merged**